ingestion/src/metadata/profiler/interface/pandas/profiler_interface.py

```diff
@@ -171,19 +190,23 @@ def _compute_static_metrics(
         """
         import pandas as pd  # pylint: disable=import-outside-toplevel
 
+        row_dict = {}
         try:
-            row_dict = {}
+            dataframes = [df.dropna() for df in runner]
```

Comment:
I think if we gonna drop rows with `na` we should state very clearly in the documentation and make it clear in the migration doc as this will change the count of certain metrics.

Reply:
Yes, will make that clear